### ISSUE TITLE
Public repos shouldn't grant collaborators pull permissions

### ISSUE THREAD

**Issue by terrajobst** [dotnet/org-policy#8]
This used to be valuable so that we can assign issues, but that's no longer necessary (you can assign issues to anyone with permissions for the repo *and* anyone who commented on it). The only downside is the loss of the "collaborator" badge, but the downside for admins (having a huge list to deal with) is more important to fix IMHO.

**Comment by jeffwilcox** [comment]
Would you recommend this company-wide?

**Comment by terrajobst** [comment]
Yes. The challenge is that virtually all repos have this and it's a PITA to address. There were also issues with allowing contributors to interact with our CI system via commands. But I think those should be managed differently.
